decidim-core/app/presenters/decidim/home_stats_presenter.rb

```diff
@@ -7,7 +7,7 @@ class HomeStatsPresenter < Rectify::Presenter
 
     # Public: Render a collection of primary stats.
     def highlighted
-      highlighted_stats = Decidim.stats.only([:users_count, :processes_count]).with_context(organization).map { |name, data| [name, data] }
+      highlighted_stats = Decidim.stats.only([:users_count, :proposals_count]).with_context(organization).map { |name, data| [name, data] }
```

Comment:
Why this change?

Reply:
Is following the design, but maybe is not necessary, I will revert it